packages/react-native/package.json

```diff
@@ -64,7 +64,7 @@
     "react-native": "0.68.2",
     "react-native-builder-bob": "^0.18.3",
     "react-native-get-random-values": "^1.8.0",
-    "react-native-webrtc": "^106.0.7",
+    "react-native-webrtc": ">=106.0.0",
```

Comment:
Do we want this? Asking since the config-plugin is only explicitly compatible with v106.x.x

Reply:
The config plugin does work fine with newer versions of react-native-webrtc up till v111.

I wanted to maximize compatible versions, even if you're not using expo. I could cap it between 106 and 111...

Reply:
> I could cap it between 106 and 111

@niravcodes I like that

Reply:
ok done!